.changeset/short-games-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/number-input': patch
+---
+
+onBlur runs with arrow buttons
```

Comment:
Can we be more explicit with the change notes? What was the problem before? How does this impact consumers? 
Also, we've been starting to add the JIRA ticket into the change notes as well

Reply:
updated!